### PR TITLE
replace `go get` with `go install`

### PR DESCRIPTION
**Pull request by samuelkarp**
The latest version of golang.org/x/sys requires Go 1.17 and cannot be compiled with older versions of Go.  github.com/vbatts/git-validation imports golang.org/x/sys and specifies a compatible version of golang.org/x/sys in its go.mod, but using `go get -u` overrides the version.  Use `go install` instead, which will respect the version specified in git-validation's go.mod.

See also https://github.com/vbatts/git-validation/issues/56 and https://github.com/opencontainers/distribution-spec/issues/343.

### CODE REVIEW

**Review by AkihiroSuda — APPROVED** [review]

File: action.yml [hunk 1]
```diff
@@ -58,9 +58,9 @@ runs:
         GO111MODULE: off
```
Comment (fuweid) [inline action.yml]: do we need to enable gomod?

Comment (akhilerm) [reply action.yml]: I think we should enable gomod by default now, as we are no longer using the `go get` functionality. I assume it was kept disabled for getting the dependencies using go get itself

Comment (samuelkarp) [reply action.yml]: > do we need to enable gomod?

Yes, thank you for catching this.  Updated.  You can see a successful run on https://github.com/samuelkarp/go-runc/pull/1.

**Review by akhilerm — APPROVED** [review]

**Review by mxpv — APPROVED** [review]

**Review by fuweid — COMMENTED** [review]
LGTM！

**Comment by samuelkarp** [comment]
I've tagged this as `v1.1.0`.

**Pull request merged**